tests/python/pants_test/engine/test_scheduler.py

```diff
@@ -85,21 +61,6 @@ def test_use_params(self):
     with self.assertRaises(Exception):
       self.scheduler.product_request(str, [Params(a)])
 
```

Comment:
This looks like a bad merge: should drop the changes in this file I think.

Reply:
Correct! Will do.